plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -1185,6 +1185,53 @@ private static function _get_subscriptions_from_product_id( $product_id, $single
 		return false;
 	}
 
+	/**
+	 * Get local plugins
+	 */
+	public static function get_local_plugins() {
+		if ( ! function_exists( 'get_plugins' ) ) {
+			require_once ABSPATH . 'wp-admin/includes/plugin.php';
+		}
+		$plugins = get_plugins();
+		
+		$output_plugins = array();
+		foreach ( $plugins as $filename => $data ) {
+			array_push($output_plugins, array(
+				'_filename'  => $filename,
+				'_type'      => 'plugin',
+				'slug'       => dirname( $filename ),
+				'Version'    => $data['Version'],
```

Comment:
I would suggest lowercase here and elsewhere.

```suggestion
				'version'    => $data['Version'],
```

Reply:
Can we keep it as is? There is existing code with _Version_ I would like to keep it consistent.
Especially ensure that `get_local_woo_plugins` and `get_local_plugins` have the same format.
Long term, I agree, we should clean up and make it all lowercase.

Reply:
Cool, I should've realised you did this with a purpose.